catalogs/catalog-common/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergPropertiesUtils.java

```diff
@@ -39,12 +39,20 @@ public class IcebergPropertiesUtils {
     map.put(IcebergConstants.WAREHOUSE, IcebergConstants.WAREHOUSE);
     map.put(IcebergConstants.CATALOG_BACKEND_NAME, IcebergConstants.CATALOG_BACKEND_NAME);
     map.put(IcebergConstants.IO_IMPL, IcebergConstants.IO_IMPL);
+    // S3
     map.put(IcebergConstants.GRAVITINO_S3_ENDPOINT, IcebergConstants.ICEBERG_S3_ENDPOINT);
     map.put(IcebergConstants.GRAVITINO_S3_REGION, IcebergConstants.AWS_S3_REGION);
     map.put(IcebergConstants.GRAVITINO_S3_ACCESS_KEY_ID, IcebergConstants.ICEBERG_S3_ACCESS_KEY_ID);
     map.put(
         IcebergConstants.GRAVITINO_S3_SECRET_ACCESS_KEY,
         IcebergConstants.ICEBERG_S3_SECRET_ACCESS_KEY);
+    // OSS
+    map.put(IcebergConstants.GRAVITINO_OSS_ENDPOINT, IcebergConstants.ICEBERG_OSS_ENDPOINT);
+    map.put(
+        IcebergConstants.GRAVITINO_OSS_ACCESS_KEY_ID, IcebergConstants.ICEBERG_OSS_ACCESS_KEY_ID);
+    map.put(
+        IcebergConstants.GRAVITINO_OSS_ACCESS_KEY_SECRET,
+        IcebergConstants.ICEBERG_OSS_ACCESS_KEY_SECRET);
```

Comment:
Do you need to do the unification the configurations for different storage firstly, or after this PR is merged?

Reply:
I prefer to merge this PR first then unify the cloud configurations.